R/rstudio.R

```diff
@@ -11,16 +11,27 @@
 #'   * Adds RStudio files to `.Rbuildignore`, if project is a package
 #'
 #' @param line_ending Line ending
+#' @param reformat If `TRUE`, the `.Rproj` is setup with common options that
+#'   reformat files on save: adding a trailing newline, trimming trailing
+#'   whitespace, and setting the line-ending. This is best practice for
+#'   new projects.
+#'
+#'   If `FALSE`, leaves all these settings alone, as is most appropriate
+#'   when you're contributing to someone else's project.
```

Comment:
```suggestion
#'   If `FALSE`, these options are left unset, which is more appropriate when
#'   you're contributing to someone else's project that does not have its own
#'   `.Rproj` file.
```

At first the wording made me think different settings were being harmonized or merged. But this is really about preventing an external contributor's preference for using RStudio from "leaking" whitespace changes into a PR.